storage_service/storage_service/settings/base.py

```diff
@@ -181,6 +189,39 @@ def get_env_variable(var_name):
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#authentication-backends
 
 AUTHENTICATION_BACKENDS = ["django.contrib.auth.backends.ModelBackend"]
+
+try:
+    PASSWORD_MINIMUM_LENGTH = int(environ.get("SS_AUTH_PASSWORD_MINIMUM_LENGTH", 8))
+except ValueError:
+    PASSWORD_MINIMUM_LENGTH = 8
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+        "OPTIONS": {"min_length": PASSWORD_MINIMUM_LENGTH},
+    }
+]
+
+# Additional password validation is enabled by default. To disable, set the
+# following AUTH_PASSWORD_* settings to false in the environment.
+if not is_false(environ.get("SS_AUTH_PASSWORD_COMMON_VALIDATION", "")):
+    AUTH_PASSWORD_VALIDATORS += [
+        {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"}
+    ]
+
+if not is_false(
+    environ.get("SS_AUTH_PASSWORD_USER_ATTRIBUTE_SIMILARITY_VALIDATION", "")
```

Comment:
`environ.get` expects a string as a key (since all env variables are strings) but I've modified it to the following to help with readability:

```python
if not is_true(environ.get("SS_AUTH_PASSWORD_DISABLE_COMMON_VALIDATION", "false")):
    ...
```

Reply:
(edited the above to point out that it's `is_true`, not `environ.get`, that's the limiting factor here)